integration/spark/src/test/scala/org/apache/carbondata/spark/util/ExternalColumnDictionaryTestCase.scala

```diff
@@ -205,6 +205,20 @@ class ExternalColumnDictionaryTestCase extends QueryTest with BeforeAndAfterAll
       loadSqlRelation, "deviceInformationId", "10086")
   }
 
+  test("COLUMNDICT and ALL_DICTIONARY_PATH can not be used together") {
+    try {
+      sql(s"""
+        LOAD DATA LOCAL INPATH "$complexFilePath1" INTO TABLE loadSqlTest
+        OPTIONS('COLUMNDICT'='$extColDictFilePath1',"ALL_DICTIONARY_PATH"='$extColDictFilePath1')
+        """)
+      assert(false)
```

Comment:
write assert(false) here is just to make sure that flow goes to catch part, you know, if not go to catch part, then test case should fail. ...i think it is necessary...